blockbase/styles/featured-one.json

```diff
@@ -0,0 +1,36 @@
+{
+	"version": 2,
+	"title": "Featured One",
```

Comment:
@beafialho should we rename these color palettes/variations?

Reply:
Are these the Videomaker ones?

Reply:
These are the Blockbase color palettes from the customizer converted to theme variations.  They didn't have names in Blockbase, just pictures and I used the names that we had given them in configuration.  As user-surfaced names they are certainly lack-luster.

<img width="337" alt="image" src="https://user-images.githubusercontent.com/146530/172621837-611b2164-c3bf-4bca-9025-28c6d4596b33.png">

Reply:
Oh, I see! Here's what I suggest then:

Default > **Ocean Breeze**
Featured (Red) > **Ruby Wine**
Featured (Green) > **Rainforest**
Featured (Pastel Colors) > **Charcoal**

Reply:
Thanks!  Nice names.  :)

Reply:
<img width="276" alt="image" src="https://user-images.githubusercontent.com/146530/172887172-124b60d3-f1f0-4bdf-94ac-393cea79e14f.png">
<img width="268" alt="image" src="https://user-images.githubusercontent.com/146530/172887229-edc73350-8d76-4d0a-8eec-d0d41d1893ca.png">